data/pigui/views/mainmenu.lua

```diff
@@ -42,7 +42,7 @@ local max_flavours = 22
 local startLocations = {
 	{['name']=lui.START_AT_MARS,
 	 ['desc']=lui.START_AT_MARS_DESC,
-	 ['location']=SystemPath.New(0,0,0,0,18),
+	 ['location']=SystemPath.New(0,0,0,0,22),
```

Comment:
There's an instance of '0,0,0,0,18' in /src/main.cpp also. For starting from the command line.